examples/inverse/mixed_source_space_inverse.py

```diff
@@ -26,21 +25,21 @@
 # Set dir
 data_path = mne.datasets.sample.data_path()
 subject = 'sample'
-data_dir = op.join(data_path, 'MEG', subject)
-subjects_dir = op.join(data_path, 'subjects')
-bem_dir = op.join(subjects_dir, subject, 'bem')
+data_dir = data_path / 'MEG' / subject
+subjects_dir = data_path / 'subjects'
+bem_dir = subjects_dir / subject / 'bem'
 
 # Set file names
-fname_mixed_src = op.join(bem_dir, '%s-oct-6-mixed-src.fif' % subject)
-fname_aseg = op.join(subjects_dir, subject, 'mri', 'aseg.mgz')
+fname_mixed_src = bem_dir / f'{subject}-oct-6-mixed-src.fif'
+fname_aseg = subjects_dir / subject / 'mri' / 'aseg.mgz'
 
-fname_model = op.join(bem_dir, '%s-5120-bem.fif' % subject)
-fname_bem = op.join(bem_dir, '%s-5120-bem-sol.fif' % subject)
+fname_model = bem_dir / f'{subject}-5120-bem.fif'
+fname_bem = bem_dir / f'{subject}-5120-bem-sol.fif'
 
-fname_evoked = data_dir + '/sample_audvis-ave.fif'
-fname_trans = data_dir + '/sample_audvis_raw-trans.fif'
-fname_fwd = data_dir + '/sample_audvis-meg-oct-6-mixed-fwd.fif'
-fname_cov = data_dir + '/sample_audvis-shrunk-cov.fif'
+fname_evoked = data_dir / '/sample_audvis-ave.fif'
+fname_trans = data_dir / '/sample_audvis_raw-trans.fif'
+fname_fwd = data_dir / '/sample_audvis-meg-oct-6-mixed-fwd.fif'
+fname_cov = data_dir / '/sample_audvis-shrunk-cov.fif'
```

Comment:
for these four lines, the leading slash that is *inside* the quotation marks can go away. So, e.g., `'/sample_audvis-ave.fif'` can be `'sample_audvis-ave.fif'`

Reply:
Also, the doc build is failing because `mne.setup_volume_source_space` doesn't accept path-like instances for 3 of its arguments, only str.

Reply:
> Also, the doc build is failing

thanks for the tip. I'll open an issue about it.